### ISSUE TITLE
Run v2: Confusing error message when hitting bad URL

### ISSUE THREAD

**Issue by jbonisteel** [ooni/run#179]
I have been testing out what happens if you try and load a bad/incorrect Run v2 link. Here is the link I was using: https://run.test.ooni.org/v2/104579

I have attached a screen recording of what happens. 

@aanorbel what options do we have for improving the user experience? Can we improve/change the copy of the error message that appears? Right now it just says 'error' - which isn't very informative. If you can let me know what you think, we can decide how we might want to improve this 

https://github.com/ooni/run/assets/15895520/734a40e8-a59d-46bb-9362-1602664e187f

**Comment by aanorbel** [comment]
I agree. During development, I used Error as a place holder knowing that the copy would be changed as well as some of the other copy in the app.

**Comment by aanorbel** [comment]
Error Message would be changed after copy is reviewed

**Comment by hellais** [comment]
We should discuss this with @sloncocs and @agrabeli to come up with a better error message
